docs/getting-started/creating-users.md

```diff
@@ -0,0 +1,39 @@
+## Auth Concepts
+
+Fleet has a concept of Users, AuthCerts, Roles and Policies.
+
+AuthCerts are for authenticating users when they access their fleet to identify who they are. Your first AuthCert was creating in the [installing fleet](/getting-started/installing-fleet-tool) guide. For more info on creating and revoking your certs see [managing authentication certificates](/how-to/auth/manage-certs).
+
+Roles and Policies are for authorizing users and dictating what they can and can't do. Policies are fine grained permissions which are grouped into Roles and applied to Users. For example you might have two policies ViewEnvironments and ModifyReleases which you combine into a role Developer, and apply to particular users.
+
+The first user created will have the Admin Role, which grants all privileges. Every other user will have no Roles, thus no privileges upon creation. When you are creating a new user you need to remember to give them the necessary Roles.
+
+### Creating a User
+
+Create a new user like so:
+
+```
+$ fleet auth user add test@example.com
+```
+
+This will send a verification email to that address.
```

Comment:
Should explain/link to what happens next in this process.
Clarify that they are unable to log in until they are verified and have a certificate.